src/resources/scepter.ts

```diff
@@ -19,10 +19,12 @@ import { Potion } from "../potions";
 import { garboAverageValue, garboValue } from "../value";
 import { canAdventure, canEquip, Item, myLevel, myMeat, Skill, toSlot, useSkill } from "kolmafia";
 
+const FFV = globalOptions.prefs.valueOfFreeFight;
```

Comment:
We actually shouldn't fix this as a const right up top here. The way grimoire args works, this will be defined before args are filled, and we'll be referencing the default (or possibly just a null value?) value of the variable rather than the correct value